gtda/mapper/tests/test_nerve.py

```diff
@@ -12,13 +12,22 @@
 from gtda.mapper import Projection, OneDimensionalCover, make_mapper_pipeline
 
 
-@settings(deadline=5000)
-@given(X=arrays(dtype=float, unique=True,
-                elements=floats(allow_nan=False,
-                                allow_infinity=False,
-                                min_value=-1e6,
-                                max_value=1e6),
-                shape=array_shapes(min_dims=2, max_dims=2, min_side=11)))
+mapper_input = arrays(
+    dtype=float, unique=True, elements=floats(
+        allow_nan=False, allow_infinity=False,
+        min_value=-1e6, max_value=1e6
+    ),
+    shape=array_shapes(min_dims=2, max_dims=2, min_side=8, max_side=12)
+)
+
+
+hypothesis_settings = dict(
+    deadline=5000, suppress_health_check=(HealthCheck.data_too_large,)
+)
+
+
+@settings(**hypothesis_settings)
+@given(X=mapper_input)
```

Comment:
Nice refactoring!  Though I can't see how it relates to

> reduced the sizes of examples generated by hypothesis in nerve tests, by lowering the min_size parameter and adding a max_size parameter.

because I don't see these parameters in the diff.